libs/application/templates/aosh/work-accident-notification/src/lib/dataSchema.ts

```
@@ -0,0 +1,242 @@
+const isValid24HFormatTime = (value: string) => {
+  if (value.length !== 4) return false
+  const hours = parseInt(value.slice(0, 2))
+  const minutes = parseInt(value.slice(2, 4))
+  if (hours > 23) return false
+  if (minutes > 59) return false
+  return true
+}
+
+export const isValidPhoneNumber = (phoneNumber: string) => {
+  const phone = parsePhoneNumberFromString(phoneNumber, 'IS')
+  return phone && phone.isValid()
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)